ocrd_anybaseocr/cli/ocrd_anybaseocr_binarize.py

```diff
@@ -100,19 +105,17 @@ def dshow(self, image, info):
         ginput(1, self.parameter['debug'])
 
     def process(self):
-        try:
-            page_grp, self.image_grp = self.output_file_grp.split(',')
-        except ValueError:
-            page_grp = self.output_file_grp
-            self.image_grp = FALLBACK_IMAGE_GRP
-            LOG.info("No output file group for images specified, falling back to '%s'", FALLBACK_IMAGE_GRP)
+        assert_file_grp_cardinality(self.input_file_grp, 1)
+        assert_file_grp_cardinality(self.output_file_grp, 1)
         oplevel = self.parameter['operation_level']
 
         for (n, input_file) in enumerate(self.input_files):            
+            file_id = input_file.ID.replace(self.input_file_grp, page_grp)
             page_id = input_file.pageId or input_file.ID
             
             LOG.info("INPUT FILE %i / %s", n, page_id)
             pcgts = page_from_file(self.workspace.download_file(input_file))
+            pcgts.set_PcGtsId(file_id)
```

Comment:
```suggestion
            pcgts.set_pcGtsId(file_id)
```

Reply:
argh, looks like I lost a commit somewhere fixing those. thanks for spotting.